hello_world_on_chain/contracts/User.sol

```diff
@@ -2,33 +2,20 @@
 pragma solidity ^0.8.10;
 
 import {TAParserLib} from "../lib/TAParserLib.sol";
-import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 import {console} from "hardhat/console.sol";
 
-contract User is Ownable {
+contract User {
     event AttestedFunctionCallOutput(string output);
 
-    address private taSigningKeyAddress;
-
-    constructor() Ownable(msg.sender) {
-    }
-
-    function setTASigningKeyAddress(
-        bytes calldata taSigningKey
-    )
-    public onlyOwner
-    {
-        taSigningKeyAddress = TAParserLib.publicKeyToAddress(taSigningKey);
-    }
-
-    function verifyAttestedFnCallClaims(
-        string calldata taData
+    function demo(
+        bytes calldata publicKey,
+        string calldata transitiveAttestation
     )
-    public
+        public
     {
-        TAParserLib.FnCallClaims memory claims = TAParserLib.parseTA(
```

Comment:
To get us ready for moving to a library that we make available though NPM, I added some updates to the TAParserLib.  Previous iteration of the contract assumed that All TAs are fun call claims, this change allows for more diversity of TA parsing.